internal/handlers/discord/handler.go

```diff
@@ -207,6 +207,62 @@ func (h *Handler) RegisterCommands(s *discordgo.Session, guildID string) error {
 							Description: "Delete one of your characters",
 							Type:        discordgo.ApplicationCommandOptionSubCommand,
 						},
+						{
+							Name:        "equip",
+							Description: "Equip a weapon or shield",
+							Type:        discordgo.ApplicationCommandOptionSubCommand,
+							Options: []*discordgo.ApplicationCommandOption{
+								{
+									Type:        discordgo.ApplicationCommandOptionString,
+									Name:        "character_id",
+									Description: "Character ID",
+									Required:    true,
+								},
+								{
+									Type:        discordgo.ApplicationCommandOptionString,
+									Name:        "item",
+									Description: "Weapon or shield key (e.g., 'longsword', 'shield')",
+									Required:    true,
+								},
+							},
+						},
+						{
+							Name:        "unequip",
+							Description: "Unequip an item from a slot",
+							Type:        discordgo.ApplicationCommandOptionSubCommand,
+							Options: []*discordgo.ApplicationCommandOption{
+								{
+									Type:        discordgo.ApplicationCommandOptionString,
+									Name:        "character_id",
+									Description: "Character ID",
+									Required:    true,
+								},
+								{
+									Type:        discordgo.ApplicationCommandOptionString,
+									Name:        "slot",
+									Description: "Slot to unequip (main-hand, off-hand, body)",
+									Required:    true,
+									Choices: []*discordgo.ApplicationCommandOptionChoice{
+										{Name: "Main Hand", Value: "main-hand"},
+										{Name: "Off Hand", Value: "off-hand"},
+										{Name: "Body", Value: "body"},
+									},
+								},
+							},
+						},
+						{
+							Name:        "inventory",
+							Description: "Show character's inventory",
+							Type:        discordgo.ApplicationCommandOptionSubCommand,
+							Options: []*discordgo.ApplicationCommandOption{
+								{
+									Type:        discordgo.ApplicationCommandOptionString,
+									Name:        "character_id",
```

Comment:
generally thinking we do not want commands that requires things like a character id. we should provide navigation